buf/internal/toolchain.bzl

```diff
@@ -52,13 +52,13 @@ _buf_toolchain = rule(
 
 def declare_buf_toolchains(os, cpu, rules_buf_repo_name):
     for cmd in ["buf", "protoc-gen-buf-lint", "protoc-gen-buf-breaking"]:
-        ext = ""
+        exe_suffix = ""
```

Comment:
ultra nit: `cmd_suffix` would be clearer.

Reply:
done